src/applications/disability-benefits/526EZ/tests/schema/initialData.js

```diff
@@ -44,17 +44,33 @@ export default {
       specialIssues: [
         {
           code: 'TRM',
-          name: 'Personal Trauma PTSD'
-        }
-      ]
+          name: 'Personal Trauma PTSD',
+        },
+      ],
+      // providerFacility: [
```

Comment:
Does this commented section need to stay as well?

Reply:
Thank you! It's been deleted.